### PR TITLE
Base activity, timestamp saving to sharedpreferences

### PR DESCRIPTION
**Pull request by zsofinnemeth**

### CODE REVIEW

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/BaseActivity.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_base);
+    }
+
+    @Override
+    public void onPause() {
+        super.onPause();
+
+        Long timestamp = System.currentTimeMillis()/1000;
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/BaseActivity.java]: @zsofinnemeth dont need to divide by 1000

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/BaseActivity.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.support.v7.app.AppCompatActivity;
+
+import com.greenfox.fuchsit.zerdareader.R;
+
+public abstract class BaseActivity extends AppCompatActivity {
+
+    SharedPreferences timePreferences;
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/BaseActivity.java]: @zsofinnemeth this is a default sharedpreferences, not just the time can be stored in here, so name it like that

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/BaseActivity.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    }
+
+    @Override
+    public void onPause() {
+        super.onPause();
+
+        Long timestamp = System.currentTimeMillis()/1000;
+        timePreferences = PreferenceManager.getDefaultSharedPreferences(BaseActivity.this);
+        final SharedPreferences.Editor editor = timePreferences.edit();
+        editor.putLong("timestamp", timestamp);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/BaseActivity.java]: @zsofinnemeth all these lines could be in one line (timestamp variable never used again, editor never used again)

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/BaseActivity.java (hunk 3 above)
Comment (zsofinnemeth) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/BaseActivity.java]: the editor is used, so I could chop down only one line

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/BaseActivity.java (hunk 1 above)
Comment (zsofinnemeth) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/BaseActivity.java]: done, sorry

File: app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/BaseActivity.java (hunk 2 above)
Comment (zsofinnemeth) [reply app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/BaseActivity.java]: done this on a different branch and forgot about it eventually.. corrected it.

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**